remo/featuredrep/templates/featuredrep_alter.jinja

```diff
@@ -33,7 +33,7 @@
         </span>
         {{ field_errors(form.users) }}
       </div>
-      <div class="large-12 columns">
+      <div class="large-6 columns">
```

Comment:
can we leave this at 12? Then it's easier to proof-read the text. I guess the date can be moved up so it shares the space with the reps list (so, reps list = 6, date field = 6, textarea = 12). What do you think?

Reply:
The screen that I made is for the new interface.
This was before so I was trying to improve the ui
![screenshot_20180316_121806](https://user-images.githubusercontent.com/403283/37518115-20ebdcfe-2914-11e8-9a17-6a8bedfb7b89.png)